crates/cloudproof/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cloudproof"
-version = "2.1.0"
+version = "2.1.1"
```

Comment:
Same here

Reply:
in the next release, we need to publish the new cloudproof crate (with refreshed deps)